pkg/component/etcd/statefulset/statefulset.go

```diff
@@ -403,7 +403,7 @@ func (c *component) createOrPatch(ctx context.Context, sts *appsv1.StatefulSet,
 		if err != nil {
 			return err
 		}
-
+		podLabels := utils.MergeStringMaps(make(map[string]string, 0), c.values.PodAdditionalLabels, c.values.Labels)
```

Comment:
Maybe this is a redundant variable
Can we directly use `utils.MergeStringMaps(....` [here](https://github.com/gardener/etcd-druid/pull/559/files#diff-aa06cd16d8c4ce540fe8d4bdd8b404d5f7270cb99c90c3481392fcd629533d3bR421)?

Reply:
Fixed - https://github.com/gardener/etcd-druid/pull/559/commits/539e7d066f052a87c8ee5bec719c010d860e2d4d